Source/ARTConstants.m

```diff
@@ -0,0 +1,13 @@
+//
+//  ARTConstants.m
+//  Ably
+//
+//  Created by Ricardo Pereira on 23/08/2018.
+//  Copyright © 2018 Ably. All rights reserved.
+//
+
+#import "ARTConstants.h"
+
+NSString *const ARTHttpHeaderFieldErrorCodeKey = @"X-Ably-Errorcode";
+NSString *const ARTHttpHeaderFieldErrorMessageKey = @"X-Ably-Errormessage";
```

Comment:
I am guessing we can move other constants in here once this is merged.

Reply:
Yes, exactly. I think it makes more sense to have a `ARTConstants.h` file instead putting everything into `ARTTypes.h`.